### PR TITLE
UX-761/Support viewing logs for pods with multiple containers

### PR DESCRIPTION
**Pull request by knnguy**
Added a new 'containers' column to the pods list table that lists out the container names and the links to view the logs for the containers

Screen Shot 2021-01-05 at 5 13 24 PM

### CODE REVIEW

**Comment by knnguy** [comment]
This is not ready for review yet. I've decided to make some more changes to the code I wrote

File: src/app/plugins/kubernetes/components/pods/PodsListPage.js [hunk 1]
```diff
@@ -67,26 +72,41 @@ const ListPage = ({ ListContainer }) => {
...
   }
 }
-const openLogsWindow = ({ name, id, clusterName, clusterId, logs }) => () => {
-  trackEvent('View Pod Logs', { name, id, clusterName, clusterId })
-  window.open(logs)
+const openLogsWindow = (log, pod) => () => {
+  const { name, id, clusterName, clusterId } = pod
+  const containerName = log.containerName
+  trackEvent('View Pod Logs', { name, id, clusterName, clusterId, containerName })
+  window.open(log.url)
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/pods/PodsListPage.js:79]: This doesn't open with a new tab from what I remember. shouldn't you include the target when calling open? `window.open(log.url, '_blank')`

Comment (knnguy) [reply src/app/plugins/kubernetes/components/pods/PodsListPage.js:79]: Oh, I specified that in SimpleLink

```
<SimpleLink target="_blank" rel="noopener" onClick={openLogsWindow(log, pod)}>
     View Container Logs
 </SimpleLink
```

Comment (jruddell) [reply src/app/plugins/kubernetes/components/pods/PodsListPage.js:79]: ah it must still honor the default behavior then and open a new tab?

**Review by jruddell — APPROVED** [review]

**Pull request merged**